brats/models.py

```diff
@@ -5,24 +5,26 @@
 class UNet3DBlock(nn.Module):
     def __init__(self, in_channels, out_channels):
         super().__init__()
-        self.conv1 = nn.Conv3d(
-            in_channels=in_channels,
-            out_channels=out_channels,
-            kernel_size=3,
-            padding=1,
-            bias=False
-        )
-        self.conv2 = nn.Conv3d(
-            in_channels=out_channels,
-            out_channels=out_channels,
-            kernel_size=3,
-            padding=1,
-            bias=False
-        )
+        self.conv1 = nn.Conv3d(in_channels=in_channels,
+                               out_channels=out_channels,
+                               kernel_size=3,
+                               padding=1,
+                               bias=False)
+        self.bn1 = nn.BatchNorm3d(out_channels)
+        self.conv2 = nn.Conv3d(in_channels=out_channels,
+                               out_channels=out_channels,
+                               kernel_size=3,
+                               padding=1,
+                               bias=False)
+        self.bn2 = nn.BatchNorm3d(out_channels)
+        self.act = nn.PReLU()
```

Comment:
What is the reason for having only one activation function at the end instead of having one activation after each convolution? Isn't the latter more common approach?

Reply:
It was based on other code, however I checked how Kuba, Pablo and Michał did it and changed it to activations after each convolution.

Reply:
Great!

Reply:
Additional comment: I changed it to two separate instances of PReLu, because of P in PReLu - it wouldn't work as expected if it would be 1 instance 😅

Reply:
Yep, good catch.